app/workers/education_form/templates/secondary_contact.erb

```diff
@@ -1,6 +1,6 @@
 Name, Address, & Telephone Number of Contact:
         <%= @applicant.secondaryContact&.fullName %>
-        <%- if @applicant.secondaryContact&.sameAddressAndPhone -%>
+        <%- if @applicant.secondaryContact&.sameAddress || @applicant.secondaryContact&.sameAddressAndPhone -%>
```

Comment:
sameAddressAndPhone doesn't exist in any of the schemas, i think we should remove it